manifests/plugins_dir.pp

```diff
@@ -14,7 +14,7 @@
       recurse => $recurse,
       purge   => $purge,
       force   => $force,
-      require => Package['sensu'],
+      require => Package[$::sensu::package::package_name],
     }
```

Comment:
I'm not seeing where the package_name is changed at all - could you help me with what this is for?